.unimportedrc.json

```
@@ -0,0 +1,23 @@
+{
+  "preset": "next",
+  "entry": [
+    "./app/**/*.{js,jsx,ts,tsx}",
+    "./layouts/**/*.{js,jsx,ts,tsx}",
+    "./nextx.mdx.use.mjs"
+  ],
+  "extensions": [".js", ".jsx", ".ts", ".tsx"],
+  "flow": false,
+  "ignorePatterns": [
+    "**/node_modules/**",
+    "**/*.tests.{js,jsx,ts,tsx}",
+    "**/*.test.{js,jsx,ts,tsx}",
+    "**/*.spec.{js,jsx,ts,tsx}",
+    "**/tests/**",
+    "**/__tests__/**",
```

```suggestion
    "**/__tests__/*.test.mjs",
```

in according with [`jest.conifg.mjs`](https://github.com/nodejs/nodejs.org/blob/main/jest.config.mjs)

---

Actually this file was commited accidentally hehe!